datastore-cli/src/main/scala/org/allenai/datastore/cli/Common.scala

```diff
@@ -1,9 +1,23 @@
 package org.allenai.datastore.cli
 
+import org.allenai.datastore.Datastore
+
+import scala.util.{ Try, Failure, Success }
+
 object Common {
   def printDefaultDatastoreWarning() = {
     System.err.println("No datastore explicitly specified.")
     System.err.println("The default (public) datastore will be used.")
     System.err.println("To specify another datastore (such as 'private'), use `-d private`.")
   }
+
+  def handleDatastoreExceptions[T](f: => T): Try[T] = {
```

Comment:
It's just a suggestion. It's a weird function as-is - you're returning a `Try`, but have an embedded `try` statement, which is odd.

It also only captures `Datastore.DSException`, and will throw any others encountered, which isn't what I'd expect when calling it.